### ISSUE TITLE
FR: V2 Recommendations block with video 

### ISSUE THREAD

**Issue by ZoeBeykirch** [aemsites/vg-macktrucks-com#72]
### Context/Description

As a Mack Trucks content author, I want to add video content into the V2 Recommendations block so that I can enrich the recommended bulldog magazine articles and other web pages with more engaging multimedia content that will enhance the user experiences. 

### Acceptance Criteria

This ticket should be picked up after https://github.com/hlxsites/vg-macktrucks-com/issues/750

**User AC:**
- [x] As a website user, I expect to see thumbnail video clips in the recommended article section while I'm reading a Bulldog Magazine article. 

**Editor AC:**
- [x] As a content editor, I can choose to use either an image or a video for the recommended article section.
- [x] As a content editor, I can use this block from the AEM Sidekick Library. 

**Non-functional AC:**
- [x] The block can use a 30 second video clip without sound. 
- [x] The block will use custom controls: play/pause
- [x] The block will be responsive so that the display is correct across different devices and screen sizes. 
- [x] The block can be used in aemsites/vg-macktrucks-com#88
- [x] The video will come from the metadata within the article; a property `video-url` will be added to the article with the link of the video (Mp4 or AEM assets video link)
- [x] No new variant needed.
- [x] The default options for the video will be muted, loop and autoplay.

### Related tickets
- #263 
- #220 

### Out of scope

- AEM Asset integration is not required if it will cause a blocker. 
- Videos with sound, therefore no mute/unmute functionality is required. 
- Autoplay (can be handled in a separate ticket if needed).
- The videos should load efficiently without significant buffering or delays. - This has been developed in ticket #263 

### Notes
- What would be the default options? (mutted, loop, autoplay).
- Metadata property to be added - We need Urtzi for this.

### Attachment / link to Figma

Desktop: https://www.figma.com/design/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=15079-3596&m=dev

![Image](https://github.com/aemsites/vg-macktrucks-com/assets/125895595/33399110-958d-402c-a546-11b924d50020)

Mobile: https://www.figma.com/design/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?m=dev&node-id=2222-52924

![Image](https://github.com/aemsites/vg-macktrucks-com/assets/125895595/9e15226a-8bb1-4b30-9f58-e96e23971c2c)

Library
https://main--vg-macktrucks-com--volvogroup.aem.page/block-library/blocks/v2-recommendations

**Comment by ZoeBeykirch** [comment]
This should be unblocked after recent release, let's re-investigate

**Comment by Meli-lazarte** [comment]
@WendyKruger which will be the default options for the video? (muted, loop, autoplay)

**Comment by Meli-lazarte** [comment]
Three options apply.

**Comment by alan-le-bihan** [comment]
Removing from the sprint as the BE part should be done first.
